packages/puppeteer-core/src/api/BrowserContext.ts

```diff
@@ -63,25 +64,23 @@ export interface BrowserContextEvents extends Record<EventType, unknown> {
 }
 
 /**
- * BrowserContexts provide a way to operate multiple independent browser
- * sessions. When a browser is launched, it has a single BrowserContext used by
- * default. The method {@link Browser.newPage | Browser.newPage} creates a page
- * in the default browser context.
+ * {@link BrowserContext} represents individual sessions within a
+ * {@link Browser | browser}.
  *
- * @remarks
+ * When a {@link Browser | browser} is launched, it has a single
+ * {@link BrowserContext | browser context} by default. Others can be created
+ * using {@link Browser.createIncognitoBrowserContext}. These "incognito"
+ * {@link BrowserContext | browser contexts} don't write any browsing data to
```

Comment:
I learned that it might or might not be true regarding writing to disk (especially, cross-browser)

Reply:
Hmm, I'll just remove it then.